### PR TITLE
fix(AddAccount): Derivation path drop down is empty

### LINKED ISSUE #16635: Derivation path drop down is empty
# Bug Report

## Steps to reproduce

Image

```
DBG 2024-10-28 15:46:32.187Z NewBE_callPrivateRPC                       topics="rpc" tid=33986 file=core.nim:27 rpc_method=wallet_getDerivedAddresses
WRN 2024-10-28 15:46:32.273Z qt warning                                 topics="qt" tid=33962 text="QQuickItem::stackBefore: Cannot stack QQuickRectangle(0x5d4b434089d0, parent=0x5d4b4356ee70, geometry=0,0 448x0) before QQuickRepeater(0x5d4b436bf090), which must be a sibling" file=:0 category=default
DBG 2024-10-28 15:46:32.275Z [threadpool task thread] initiating task   topics="task-threadpool" tid=33987 file=threadpool.nim:56 messageType=FetchDetailsForAddressesTaskArg:ObjectType threadid=33987 task="{\"$type\":\"FetchDetailsForAddressesTaskArg:ObjectType\",\"uniqueId\":\"1730130392\",\"addresses\":[\"0xe54e9305e399b74ab9ad5e5cded35ab9b162b6bc\"],\"vptr\":102577662550608,\"slot\":\"onAddressDetailsFetched\"}"
DBG 2024-10-28 15:46:32.275Z NewBE_callPrivateRPC                       topics="rpc" tid=33987 file=core.nim:27 rpc_method=wallet_addressExists
DBG 2024-10-28 15:46:32.275Z NewBE_callPrivateRPC                       topics="rpc" tid=33987 file=core.nim:27 rpc_method=wallet_addressDetails
WRN 2024-10-28 15:46:33.583Z qt warning                                 topics="qt" tid=33962 text="QQuickItem::stackBefore: Cannot stack QQuickRectangle(0x5d4b43226250, parent=0x5d4b4356ee70, geometry=0,0 448x0) before QQuickRepeater(0x5d4b436bf090), which must be a sibling" file=:0 category=default
```

### Additional Information

- Status desktop version: master

### PR DESCRIPTION
**Pull request by caybro**
### What does the PR do

- use `StatusListView` for the popup/menu contentItem
- add the menu to SB too

Fixes #16635

### Affected areas

DerivationInput

### Architecture compliance

- [x] I am familiar with the application architecture and agreed good practices.
My PR is consistent with this document: [Status Desktop Architecture Guide](https://github.com/status-im/status-desktop/blob/master/CONTRIBUTING.md)

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://github.com/user-attachments/assets/3151eca5-513d-475e-8035-4a72d48131ac

### CODE REVIEW

File: ui/imports/shared/popups/addaccount/panels/DerivationPathInput.qml [hunk 1]
```diff
@@ -107,7 +107,7 @@ Item {
         readonly property var passthroughKeys: [Qt.Key_Left, Qt.Key_Right, Qt.Key_Escape]
 
         Shortcut {
-            sequence: StandardKey.Copy
+            sequences: [StandardKey.Copy]
```
Comment (caybro) [inline ui/imports/shared/popups/addaccount/panels/DerivationPathInput.qml:110]: This was yielding a warning too about the sequence being ambiguos -> the shortcut wasn't functional

**Comment by caybro** [comment]
@alaibe @jrainville do we want this fix in 2.31 too?

**Comment by anastasiyaig** [comment]
> @alaibe @jrainville do we want this fix in 2.31 too?

its not broken in 2.31 yet it seems

Screenshot 2024-10-29 at 14 42 20

**Comment by caybro** [comment]
Cool... I really wonder what recent change broke this

**Review by alaibe — APPROVED** [review]

**Review by saledjenic — APPROVED** [review]

File: ui/imports/shared/popups/addaccount/panels/DerivationPathSelection.qml [hunk 2]
```diff
@@ -12,37 +12,34 @@ StatusMenu {
...
+            components: [
+                StatusIcon {
+                    visible: root.selectedRootPath === root.roots[index]
+                    icon: "checkmark"
+                    color: Theme.palette.primaryColor1
                 }
+            ]
+
+            onClicked: {
+                root.selected(root.roots[index])
```
Comment (alexjba) [inline ui/imports/shared/popups/addaccount/panels/DerivationPathSelection.qml:41]: 😄 Nice naming

**Review by alexjba — APPROVED** [review]

**Pull request merged**